components/log-viewer-webui/client/src/pages/IngestPage/IngestState/QueryJobsResp.ts

```
@@ -0,0 +1,128 @@
+        "M",
+        "G",
+        "T",
+        "P",
+        "E",
+        "Z"];
+
+    for (let i = 0; i < siPrefixes.length; ++i) {
+        if (BYTES_PER_KIBIBYTE > Math.abs(num)) {
+            return `${Math.round(num)} ${siPrefixes[i]}B`;
+        }
+        num /= BYTES_PER_KIBIBYTE;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)